lib/ramble/ramble/modifier.py

```diff
@@ -150,10 +150,14 @@ def modded_variables(self, app, extra_vars={}):
                 else:
                     prev_val = ""
 
+                sep = ""
```

Comment:
nit: I prefer the single-assignment form:

```
if prev_val != "" and prev_val is not None:
  sep = var_mod["separator"]
else:
  sep = ""
```

Reply:
Thanks, fixed!